pawan/html/practice_assignmenr/Programming_Exercises4.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Resume</title>
+        <style> 
+        table , th , td {
```

Comment:
need tab space to this block, also we don't give space before comma,
is should look like, "table, th, td {"